frontend/.gitignore

```diff
@@ -21,3 +21,10 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+# my additions
+notes.md
```

Comment:
What is the rationale behind ignoring this file? I propose to at least use some hidden file or something like `notes.local.md`.

Reply:
OK, I will name it as suggested. This is a local file for my notes.